### ISSUE TITLE
Default to podcast image if no episode image found

### ISSUE THREAD

**Issue by Explosion-Scratch** [akhilrex/podgrab#295]
Most podcasts have an image for them, but many episodes do not:

Podcast image is shown here:
<img width="1093" alt="image" src="https://github.com/akhilrex/podgrab/assets/61319150/a9a222fc-7614-4252-b6f8-030f7d066864">

Now just add it as a default here instead of the gray PODGRAB:

<img width="1138" alt="image" src="https://github.com/akhilrex/podgrab/assets/61319150/d7fb0ba4-fec7-400c-8f9b-3cfcfa26a326">

**Comment by bonelifer** [comment]
I agree. Would be nice to have it default to the podcasts image if an episode image doesn't exist.
